packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -217,12 +217,21 @@ export function buildRootObject(_powers, _params, testJigSetter = undefined) {
           return mintyIssuerRecord;
         },
         mintGains: (gains, zcfSeat = undefined) => {
+          assert.typeof(
+            gains,
+            'object',
+            details`gains ${gains} must be an object`,
```

Comment:
while this is true, we can helpfully say more. 
```suggestion
            details`gains ${gains} must be an amountKeywordRecord`,
```